src/aiovodafone/api.py

```diff
@@ -244,9 +244,10 @@ def convert_uptime(self, uptime: str) -> datetime:
             seconds=int(uptime),
         )
 
-    async def login(self) -> bool:
+    async def login(self, **kwargs: dict[str, Any]) -> bool:
```

Comment:
I think there is no plan for other parameters, so I would prefer:

```suggestion
    async def login(self, force_logout: Bool = False) -> bool:
```